vllm/config/speculative.py

```diff
@@ -271,6 +294,31 @@ def __post_init__(self):
             # draft related config as None here.
             self.draft_model_config = self.target_model_config
             self.draft_parallel_config = self.target_parallel_config
+        elif self.method == "suffix":
+            if not has_arctic_inference():
+                raise ImportError(
+                    "Arctic Inference is required for suffix decoding. "
+                    "Please install via `pip install arctic-inference`.")
```

Comment:
Thanks for raising the PR, glad to see we are upstreaming this!

Since Suffix Decoding only depends on a single SuffixTree module in arctic-inference, should we require a full installation?

Curious what's vLLM team's take on this.